pandas/core/dtypes/cast.py

```diff
@@ -569,16 +569,14 @@ def coerce_to_dtypes(result, dtypes):
     if len(result) != len(dtypes):
         raise AssertionError("_coerce_to_dtypes requires equal len arrays")
 
-    from pandas.core.tools.timedeltas import _coerce_scalar_to_timedelta_type
-
     def conv(r, dtype):
         try:
```

Comment:
btw an upcoming PR is going to make the try/except part of this function unnecessary.